lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_constants.py

```diff
@@ -64,7 +64,7 @@
     "precipitation_rate": Bounds((0, 0.00003), "m s-1"),
     "rainfall_rate": Bounds((0, 0.00003), "m s-1"),
     "rainfall_rate_in_vicinity": Bounds((0, 0.00003), "m s-1"),
-    "relative_humidity": Bounds((0, 1.1), "1"),
+    "relative_humidity": Bounds((0, 2.0), "1"),
```

Comment:
We may need such a high bound over the poles, but do we have to have an upper bound in excess of every point on Earth? It rather stretches things at the top end. This comes back to your suggestion of point specific bounds as a future requirement, which I think sounds eminently sensible.

Reply:
See plots on ticket, the code will fail if it tries to put a bound on if there is data higher than the bound